app/views/container_image/_main.html.haml

```diff
@@ -3,10 +3,10 @@
  .col-sm-12.col-md-12.col-lg-6
   = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
                                                              :items => textual_group_properties}
-  = render :partial => "shared/summary/textual_multilabel", :locals => {:title => _("Packages"),
-                                                            :items => textual_group_packages}
  .col-sm-12.col-md-12.col-lg-6
   = render :partial => "shared/summary/textual", :locals => {:title => _("Relationships"),
                                                              :items => textual_group_relationships}
   = render :partial => "shared/summary/textual_tags", :locals => {:title => _("Smart Management"),
                                                                   :items => textual_group_smart_management}
+  = render :partial => "shared/summary/textual", :locals => {:title => _("Configuration"),
+                                                             :items => textual_group_configuration}
```

Comment:
@dkorn github doesn't like files that are missing the last `LF` (please remember to configure your RubyMine accordingly)